mosquitto/templates/deployment.yaml

```
@@ -38,6 +39,7 @@ spec:
               port: websocket
             initialDelaySeconds: 3
             periodSeconds: 3
+          securityContext: {{- toYaml .Values.securityContext | nindent 12 }}
```

Only render if `securityContext` is set.
```suggestion
        {{- with .Values.securityContext }}
          securityContext:
            {{ toYaml . | indent 12 | trim }}
        {{- end }}
```